zulipterminal/model.py

```diff
@@ -699,6 +700,38 @@ def can_user_edit_topic(self) -> bool:
         self.controller.report_error("User not found")
         return False
 
+    def toggle_topic_resolve_status(self, stream_id: int, topic_name: str) -> None:
+        if self.can_user_edit_topic():
+            latest_msg = self.get_latest_message_in_topic(stream_id, topic_name)
+            if latest_msg:
+                time_since_msg_sent = time.time() - latest_msg["timestamp"]
+                # ZFL < 11, community_topic_editing_limit_seconds
+                # was hardcoded as int value in secs eg. 86400s (1 day) or None
+                if self.server_feature_level is None or self.server_feature_level >= 11:
```

Comment:
See https://github.com/zulip/zulip-terminal/pull/1235#discussion_r962358962

For server 2.1, this will set edit_time_limit to None.
For server 3.0 (or ZFL 11 at least), this will set edit_time_limit to the value in the server (None or the value).
For servers between those (eg. ZFL 10), this will set edit_limit to the default, ie. 86400 - this is an edge case

My impression is that the expected server behavior was
- `old` 86400 <-- 2.1
- `<11` 86400 <-- edge case
- `>=11` 86400 or None (per server value) <-- 3.0+

This appears to generate
- `old` None
- `<11` 86400
- `>=11` 86400 or None (per server value)

Is this going to fail for 2.1? If so, could you update the test to fail first?